inc/saitam.h

```diff
@@ -1076,6 +1121,33 @@ typedef enum _sai_tam_tel_type_attr_t
      */
     SAI_TAM_TEL_TYPE_ATTR_COUNTER_SUBSCRIPTION_LIST,
 
+    /**
+     * @brief The mode of TAM telemetry type
+     *
+     * @type sai_tam_tel_type_mode_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_TAM_TEL_TYPE_ATTR_MODE,
```

Comment:
This make this chicane breaking change since this attribute will be mandatory in next release

Reply:
Should this be create only with some default value ?